pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -40,8 +40,16 @@
 
 IDENTIFYING_KEYWORDS_FILEPATH = pjoin(HERE, "identifying_keywords.json")
 
-with open(IDENTIFYING_KEYWORDS_FILEPATH) as infile:
-    IDENTIFYING_KEYWORDS = json.load(infile)
+
+@functools.lru_cache()
+def _get_default_identifying_keywords():
+    with open(IDENTIFYING_KEYWORDS_FILEPATH) as infile:
+        IDENTIFYING_KEYWORDS = json.load(infile)
+    return list(IDENTIFYING_KEYWORDS)
```

Comment:
Or, would this be better as:

```suggestion
    return tuple(IDENTIFYING_KEYWORDS)
```

That way mutation simply wouldn't be possible?

Reply:
but then we are back to the creation of the list having an implicit copy instead of explicit.
And the same issue is true for the (alternative) identifying keywords being passed in, so I needed to copy that list.

Reply:
Ah...  I see the point.  Even within the module, one would not be able to mutate the iterable.
And instead of copying a list in the "public" method, one must create a list from the tuple.
That works for me.